code/stable/gamephys/Website/Chipmunk_SRS.html

```diff
@@ -1719,6 +1726,63 @@ <h3>Data Definitions</h3>
 </tr>
 </table>
 </div>
+<div id="DD:momentOfInertia">
+<table class="ddefn">
+<tr>
+<th>Label</th>
+<td><p class="paragraph">Moment of inertia</p></td>
+</tr>
+<tr>
+<th>Symbol</th>
+<td><p class="paragraph"><em><b>I</b></em></p></td>
+</tr>
+<tr>
+<th>Units</th>
+<td><p class="paragraph">kg&sdot;m<sup>2</sup></p></td>
+</tr>
+<tr>
+<th>Equation</th>
+<td>
+<div class="equation">
+<em><b>I</b> = &sum;m<sub>j</sub>&#8239;r<sub>j</sub><sup>2</sup></em>
+</div>
+</td>
+</tr>
+<tr>
+<th>Description</th>
+<td>
+<ul class="hide-list-style-no-indent">
+<li>
+<em><b>I</b></em> is the moment of inertia (kg&sdot;m<sup>2</sup>)
+</li>
+<li>
+<em>m<sub>j</sub></em> is the mass of the j-th particle (kg)
+</li>
+<li>
+<em>r<sub>j</sub></em> is the distance perpendicular distance between the j-th particle and the angle of rotation (m)
```

Comment:
This description isn't quite right. It should say "... the distance between the j-th particle and the axis of rotation (m)".

Reply:
Thanks, updated

Reply:
This has been updated
<img width="641" alt="image" src="https://user-images.githubusercontent.com/43192745/58677416-d9a6dd00-8329-11e9-8e64-15e7f7db4646.png">